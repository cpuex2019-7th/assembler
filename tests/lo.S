main:
    li a0, hoge
    jalr zero, ra, 0
hoge:
    jalr zero, ra, 0

