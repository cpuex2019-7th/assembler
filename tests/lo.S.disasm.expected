lui a0, 0
addi a0, a0, 12
jalr zero, ra, 0
jalr zero, ra, 0
